receiver/syslogreceiver/syslog_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright -c Google LLC
```

Comment:
Needs to be `Copyright The OpenTelemetry Authors`

Reply:
it's strange, let me modify him manually, I used `make addlicense`